docs/docs/task-environment-vars.md

```diff
@@ -44,6 +44,10 @@ names through the `--env_vars_prefix` command line flag.
   was used to start this task. For example `2015-04-02T09:37:00.596Z`.
 - `MARATHON_APP_DOCKER_IMAGE` contains the value of the app definition 
   `container.docker.image` value. For example `mesosphere/marathon:latest`, `nginx` or `nginx:1.9.3`.
+- `MARATHON_APP_CPUS` contains the value of the app definition `cpus` value.
+- `MARATHON_APP_MEMORY` contains the value of the app definition `mem` value, expressed in megabytes.
+- `MARATHON_APP_DISK_SPACE`  value of the app definition `disk` value, expressed in megabytes.
+- `MARATHON_APP_LABELS` contains list of labels of the corresponding app definition. For example: `label1:value1;label2:value2`
```

Comment:
The following variables would be easier to use from shell scripts:

MARATHON_APP_LABELS=“LABEL1 LABEL2 LABEL3”
MARATHON_APP_LABEL_LABEL1=value1
MARATHON_APP_LABEL_LABEL2=value2

We would however have to make sure we convert invalid characters/spaces in the label names to underscores.